doc/en/user/source/datadirectory/configtemplate.rst

```diff
@@ -24,7 +24,27 @@ First of all to be able to use this feature, set the following flag via system v
 Then create a file called ``geoserver-environment.properties`` in the root of GeoServer's datadir. 
 This file will contain the definitions for the variables parameterized in the catalog configuration.
 
-Now edit GeoServer's configuration files of the source machine that you want to be parametric, for example let's parameterize the URL of a store 
+We can also place the ``geoserver-environment.properties`` outside the root of GeoServer's datadir. In this case the path to the  ``properties`` file must be defined in one of the following ways:
```

Comment:
The documentation should be written in the third person, not using "we" or "you"... but I see the rest of the document is already written using them... up to you to decide if leave it as is, or fix the whole page.

Reply:
fixed